### PR TITLE
Clear the dbt relations cache every time we write to it

### PR DESCRIPTION
**Pull request by chamini2**

### CODE REVIEW

**Review by turbo1912 — APPROVED** [review]

File: src/faldbt/lib.py [hunk 1]
```diff
@@ -67,7 +67,9 @@ def initialize_dbt_flags(profiles_dir: str):
 def _get_adapter(project_dir: str, profiles_dir: str) -> SQLAdapter:
     config = parse.get_dbt_config(project_dir, profiles_dir)
 
-    return adapters_factory.get_adapter(config)  # type: ignore
+    adapter: SQLAdapter = adapters_factory.get_adapter(config)  # type: ignore
+
+    return adapter
```
Comment (turbo1912) [inline src/faldbt/lib.py:72]: why is this necessary?

Comment (chamini2) [reply src/faldbt/lib.py:72]: I had modified the adapter before returning and then removed that but did not remove the "store it in a variable" code I introduced
